### DISCUSSION TITLE
Projects (beta) field to show task progress

### DISCUSSION THREAD

**Discussion by andre-wells** [community/community#7189] [Projects and Issues] [answered]
It's super easy to add/remove tasks via checkboxes, complete them and turn those tasks into other issues.  I can even see how far the issue is progressing with it's tasks.  This is great!

![image](https://user-images.githubusercontent.com/66315105/140032775-ae00613c-c320-48af-81c9-c94b2dd20df3.png)

On the new Project View, I have access to some common fields (such as labels and milestones) and I can create my own.  This is great!  But it would be really nice to have the task progress of an issue be represented in a Projects (beta) table.

![image](https://user-images.githubusercontent.com/66315105/140032894-4b49f2b2-80d4-4c08-bd0b-550f7f585c88.png)

**Comment by ddelange** [comment, marked as answer]
duplicate of https://github.com/github/feedback/discussions/7575 ?

**Reply by andre-wells** [reply]
Thanks! Some nice ideas in that one.
